docker/quickstart/docker-compose.consumers-without-neo4j.quickstart.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  datahub-gms:
+    environment:
+    - MAE_CONSUMER_ENABLED=false
+    - MCE_CONSUMER_ENABLED=false
+  datahub-mae-consumer:
+    container_name: datahub-mae-consumer
+    depends_on:
+    - kafka-setup
+    - elasticsearch-setup
+    environment:
+    - DATAHUB_GMS_HOST=datahub-gms
+    - DATAHUB_GMS_PORT=8080
+    - MAE_CONSUMER_ENABLED=true
+    - PE_CONSUMER_ENABLED=true
```

Comment:
Does this variable not have a default anywhere? Tried looking but couldn't find so I am assuming it is `null`? Do you know?

Reply:
No idea, this comes from mae-consumer `docker.env` file, possibly legacy? @jjoyce0510 do you know?

Reply:
It defaults to false because of its usage. Instead of using Spring style properties it uses System.getenv with a "true".equals(prop). This should be refactored at some point since this won't even handle case sensitivity.